src/core/SpriteAnimator.tsx

```diff
@@ -1,72 +1,163 @@
 import * as React from 'react'
```

Comment:
as in `src/core/Example.tsx`, it is good practice to enable exhaustive-deps hook:
```suggestion
/* eslint react-hooks/exhaustive-deps: 1 */
import * as React from 'react'
```

Reply:
added the instruction

Reply:
Yes now I see them (I had disabled eslint on this workspace for some reason), I tried to quickly fix them but it isn't as straightforward, some dependencies if added need quite the rework and extensive testing because they will or might cause unnecessary re-renders. I'll see if I can work on them during the weekend